ansible/roles/texlive/tasks/main.yml

```diff
@@ -0,0 +1,46 @@
+- name: texlive | prereq
+  yum: name={{ item }} state=latest
+  with_items:
+    - perl-Tk
+    - perl-Digest-MD5
+
+- name: texlive | create the temp install folder
+  file: path="{{ texlive_dest }}/tmp/" state=directory mode=0755
+    
+- name: texlive | download and extract installer
+  unarchive:
+    copy: no
+    # Checking whether this version of texlive is installed
+    # before downloading the installer.
+    # Can the 'unarchive' test for a directory?
+    creates: "{{ TEXDIR }}"
```

Comment:
Check for something that's present in the final installed texlive instead? e.g. `/opt/texlive/2015/bin/x86_64-linux/pdftex`